src/nwb_datajoint/cli/cli.py

```diff
@@ -0,0 +1,292 @@
+                'removal_window_ms': 1.0,
+            }
+        }, sort_keys=False)
+        print(a)
+        return
+
+    import nwb_datajoint.common as ndc
+    with open(yaml_file_name, 'r') as f:
+        x = yaml.safe_load(f)
+    ndc.ArtifactDetectionParameters.insert1(x)
+
+@click.command(help="List artifact detection parameters.")
+def list_artifact_detection_parameters():
+    import nwb_datajoint.common as ndc
+    a = ndc.ArtifactDetectionParameters & {}
+    print(a)
+
+@click.command(help="Insert spike sorting recording selection")
+@click.argument('yaml_file_name', required=False)
+def insert_spike_sorting_recording_selection(yaml_file_name: Union[str, None]):
+    if yaml_file_name is None:
+        print('You must specify a yaml file. Sample content:')
+        print('==========================================')
+        a = yaml.safe_dump({
+            'nwb_file_name': 'FileName_.nwb',
+            'sort_group_id': 0,
+            'sort_interval_name': 'sort_interval_name',
+            'preproc_params_name': 'default',
+            'interval_list_name': 'interval_list_name',
+            'team_name': 'TeamName'
+        }, sort_keys=False)
+        print(a)
+        return
+
+    import nwb_datajoint.common as ndc
+    with open(yaml_file_name, 'r') as f:
+        x = yaml.safe_load(f)
+    ndc.SpikeSortingRecordingSelection.insert1(x)
+
+@click.command(help="List spike sorting recording selections for a session.")
+@click.argument('nwb_file_name')
+def list_spike_sorting_recording_selections(nwb_file_name):
+    import nwb_datajoint.common as ndc
+    a = ndc.SpikeSortingRecordingSelection & {'nwb_file_name': nwb_file_name}
+    print(a)
+
+@click.command(help="Create a spike sorting recording")
+@click.argument('yaml_file_name', required=True)
+def create_spike_sorting_recording(yaml_file_name: str):
+    import nwb_datajoint.common as ndc
+    with open(yaml_file_name, 'r') as f:
+        x = yaml.safe_load(f)
+    ndc.SpikeSortingRecording.populate([(ndc.SpikeSortingRecordingSelection & x).proj()])
+
+@click.command(help="List spike sorting recordings for a session.")
+@click.argument('nwb_file_name')
+def list_spike_sorting_recordings(nwb_file_name):
+    import nwb_datajoint.common as ndc
+    a = ndc.SpikeSortingRecording & {'nwb_file_name': nwb_file_name}
+    print(a)
+
+@click.command(help="Create a spike sorting recording view")
+@click.argument('yaml_file_name', required=True)
+def create_spike_sorting_recording_view(yaml_file_name: str):
+    import nwb_datajoint.common as ndc
+    import nwb_datajoint.figurl_views as ndf
+    with open(yaml_file_name, 'r') as f:
+        x = yaml.safe_load(f)
+    a = ndf.SpikeSortingRecordingView.populate([(ndc.SpikeSortingRecording & x).proj()])
+    print(a)
+
+@click.command(help="Insert spike sorter parameters")
+@click.argument('yaml_file_name', required=False)
+def insert_spike_sorter_parameters(yaml_file_name: Union[str, None]):
+    if yaml_file_name is None:
```

Comment:
This is probably better handled with a try...except

Reply:
I think an if statement is most appropriate here, because if there is an exception when yaml_file_name is not None, we'd want to see the exception, and not print the usage instructions.